docs/architecture/PROCESS.md

```diff
@@ -0,0 +1,47 @@
+# ADR Creation Process
+
+1. Copy the `adr-template.md` file. Use the following filename pattern: `adr-next_number-title.md`
+1. Create a draft Pull Request if you want to get an early feedback.
+1. Make sure the context and a solution is clear and well documented.
+1. Add an entry to a list in the [README](./README.md) file.
+1. Create a Pull Request to propose a new ADR.
+
+
+## ADR life cycle
+
+ADR creation is an **iterative** process. Instead of trying to solve all decisions in a single ADR pull request, we MUST firstly understand the problem and collect a feedback through a GitHub Issue.
+
+1. Every proposal SHOULD starts with a new GitHub Issue or be a result of existing Issues. The issue should contain just a brief proposal summary.
+
+1. Once the motivation is validated, a GitHub Pull Request (PR) is created with a new document based on the `adr-template.md`.
+
+1. An ADR doesn't have to arrive to `master` with an _accepted_ status in a single PR. If the motivation is clear and the solution is sound, we SHOULD be able to merge it and keep a _proposed_ status. It's preferable to have an iterative approach rather than long, not merged Pull Requests.
+
+1. If a _proposed_ ADR is merged, then it should clearly document outstanding issues either in ADR document notes or in a GitHub Issue.
+
+1. The PR SHOULD always be merged. In the case of a faulty ADR, we still want to  merge it with a _rejected_ status. The only time the ADR SHOULD NOT be merged is if the author abandons it.
+
+
+### ADR status
```

Comment:
is there a way to distinguish implemented ADRs vs not implemented?  RN there are a lot of them that haven't and I don't think the PROPOSED status reflects this per se.

Reply:
yes, good point. In the template there is a `Implemented | Not Implemented` attribute. I added a description about it here.